task3.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "fmt"
+
+func average(arr [6]int) float64 {
+  var sum int
+  for _, a := range arr {
+    sum += a
+  }
+  return float64(sum) / float64(len(arr))
+}
+
+func max (sl []string) string {
+  min, ind := 0, 0
```

Comment:
Too many unnamed variables. It's difficult to control the code flow. Please find proper names for them.